rust/connlib/clients/apple/src/lib.rs

```diff
@@ -191,21 +194,28 @@ impl WrappedSession {
             .enable_all()
             .build()
             .map_err(|e| e.to_string())?;
+        let _guard = runtime.enter(); // Constructing `PhoenixChannel` requires a runtime context.
```

Comment:
Should be fine since the portal connection starts early and it will be very obvious in the first minute of any smoke test if this line were to go missing